### ISSUE TITLE
Hamburger menu lacks expected attributes

### ISSUE THREAD

**Issue by Woebin** [JohnXLivingston/peertube-plugin-livechat#731]
**Describe the bug**
This is a violation of the [Web Content Accessibility Guidelines (WCAG)](https://www.w3.org/WAI/standards-guidelines/wcag/) [SC 4.1.2 (A) Name, Role, Value](https://www.w3.org/WAI/WCAG22/Understanding/name-role-value).

Components need to have appropriate `role` attributes to work correctly with assistive technologies.

The hamburger menu button next to each chat message opens up a menu with two options, "Copy" and "Quote". It does not have the `role="menu"` attribute though, nor do the options inside have the `role="menuitem"` attribute.

If appropriate roles are used, keyboard navigation and focus handling tend to work out of the box, as well as use with assistive technologies such as screen readers.

Pictured below is the button and menu along with Safari's developer tools.
![The menu button shown in Safari's developer tools.](https://github.com/user-attachments/assets/cfe27619-c4e8-4c91-99ac-9cb73ef1a69a)

**Expected behavior**
The button to open the menu and the menu itself should be fully accessible via keyboard as well as assistive technologies, which is achieved by correctly using appropriate roles and attributes. Please see the [MDN documentation on role="menu"](https://developer.mozilla.org/en-US/docs/Web/Accessibility/ARIA/Reference/Roles/menu_role).

**Server:**
 - Peertube version: 7.2.1
 - peertube-plugin-livechat version: 14.0.0

**Desktop:**
 - OS: MacOS 15.5
 - Browser: Safari 18.5

**Additional context**
This issue was recorded as part of an accessibility audit carried out by [Axess Lab](https://axesslab.com/), contracted by [HowlRound Theatre Commons](https://howlround.com/).

**Smartphone (please complete the following information):**

**Additional context**
